### ISSUE TITLE
USWDS - Pagination: Borders around page number have poor color contrast

### ISSUE THREAD

**Issue by jfc3-dol** [uswds/uswds#5939]
## Description

On the Pagination Page ( https://designsystem.digital.gov/components/pagination/ ), the borders around the page number have poor color contrast. The light gray ( #d1d1d1 ) on the white background in your example has a color contrast ratio of 1.5:1 or higher. I would think for those with poor vision, low vision, etc., it would be helpful to them to see how large the click or tap area is for the link. I would make the border color be at least #767676, which makes the color contrast ratio 4.5:1.

If you make changes here, you probably need to make the same changes to the code people can download and use.

**Comment by thisisdano** [comment]
I might suggest that the proper path here is to more clearly format the "buttons" as links here, by adding underlines, and considering the light border a progressive enhancement.



![Image](https://github.com/user-attachments/assets/003e71bd-2ddc-4a12-bdc6-9a6b28ea538d)

**Comment by amycole501** [comment]
It would be helpful to know which WCAG success criteria indicates the need for a touch target edge to meet 4.5:1. Your original post here doesn’t reference a WCAG success criteria.

However we can assume that[ non-text contrast (AA)](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) are the guidelines we should follow when considering the contrast of the pagination link borders.

4.5:1 states that UI elements should have a contrast ratio of at least 3:1 against adjacent colors.

However, it also states: “Boundaries: This success criterion does not require that controls have a visual boundary indicating the hit area, but if the visual indicator of the control is the only way to identify the control, then that indicator must have sufficient contrast. If text (or an icon) within a button or placeholder text inside a text input is visible and there is no visual indication of the hit area then the Success Criterion is passed."

I also agree with @thisisdano's suggestion to format the “buttons” to more resemble links (which they are).

**Comment by amyleadem** [comment]
Based on @amycole501's comment, this appears to be a usability issue rather than an accessibility issue. 

@RachelCorsino, instead of pursuing solutions to create a border color that guarantees accessible contrast, let's move forward with @thisisdano's suggestion to add `text-decoration: underline` to the interactive elements in the component. Let me know if you have any questions.
